<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5116VPI" failures="1" tests="9" name="IncubyteTest" time="425.82" errors="0" timestamp="2025-03-03T00:53:15 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="69.885"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="21.49"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="25.274"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="25.662"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="20.318"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="92.868"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="29.072"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="71.966"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="64.441">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.fail(Assert.java:96)
at com.qa.workflow.LoginAccount.disable_Account(LoginAccount.java:233)
at com.example.steps.LoginAccountStep.verify_disabled_temporarily_account_login(LoginAccountStep.java:92)
at âœ½.verify disabled temporarily account login(file:///D:/Automation/Projects/IncubyteTest/src/test/resources/feature/LoginAccount.feature:33)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- IncubyteTest -->
